WordPressAuthenticator/Authenticator/WordPressAuthenticatorConfiguration.swift

```diff
@@ -95,6 +95,16 @@ public struct WordPressAuthenticatorConfiguration {
     /// Default value is disabled
     let continueWithSiteAddressFirst: Bool
 
+    /// If enabled shows a "Sign in with site credentials" button in `GetStartedViewController` when landing in the screen after entering site address
+    ///  Used in WooCommerce iOS to enable sign in using site credentials feature.
+    ///  Disabled by default
+    let enableSiteCredentialsLoginInGetStartedScreen: Bool
+
+    /// If enabled, we will ask for WPCOM login after signing in using .org site credentials.
+    ///  WordPress iOS app can proceed without WPCOM login. But, WooCommerce iOS app needs WPCOM login.
+    ///  Disabled by default
+    let isWordPressComCredentialsRequired: Bool
```

Comment:
Nit: I'm thinking we can also clarify this further:
```suggestion
    let isWPComLoginRequiredForSiteCredentialsLogin: Bool
```

Reply:
Done in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/649/commits/a5b3874a6efcdc2eae4cdcccc4e7e97fe51e9718